data/src/client.rs

```diff
@@ -397,6 +397,12 @@ impl Map {
             .unwrap_or_default()
     }
 
+    pub fn get_our_channels<'a>(&'a self, server: &Server) -> &'a [String] {
```

Comment:
Nit: not sure `our` is needed vs `get_channels` since we never store channels that _arent_ joined.

Reply:
Good call.